chromadb/api/segment.py

```diff
@@ -189,20 +180,16 @@ def create_collection(
             )
 
         # TODO: This event doesn't capture the get_or_create case appropriately
+        # TODO: Re-enable embedding function tracking in create_collection
```

Comment:
Because the embedding function no longer gets passed down to the segment API, we can't send it along with the collection creation event. 

I'm happy to break this temporarily and fix it in a new issue - the solution is probably moving events to the client.

Reply:
When we pass the EF together with the config, we can send it again.